Tech-Lab-On-Campus/README.md

```diff
@@ -70,8 +56,24 @@ B) IDE
 * Find the Rabbitmq container and click on the URL under Port(s) for the U.I. This should open up the RabbitMQ website on your default browser.
 * Login username and password should be "guest"
 * Alternative:  Click on one of the generated URLs in your terminal, such as "http://localhost:15672/", once your docker container is up and running.
-
-6. You are now ready to start the lab. Begin by navigating to the producer folder and reading the "README.md" file. Each of the three units will contain a readme file which will give you the necessary instructions to complete the lab and test your solution. The order in which the lab should be completed as well as information regarding rabbitmq can be found below.
```

Comment:
this should be after the jupyter notebook part

Reply:
Why this is not for both ? I think this part will be done after jupyter and after ide is set.

Reply:
I changed as requested.